util/collect_metrics_files.rb

```diff
@@ -0,0 +1,253 @@
+# frozen_string_literal: true
```

Comment:
This script should probably be in a new sub-directory (i.e. 'metrics').

Reply:
The location is temporary.  It will either become some kind of task or part of a plan, and moved to whatever is the right place for it in a future PR.